include/ipv4.h

```diff
@@ -100,7 +100,7 @@ extern rte_spinlock_t ip4_stats_lock;
 #define IP4_UPD_PO_STATS(field, val) \
     do { \
         rte_spinlock_lock(&ip4_stats_lock); \
-        __IP4_ADD_STATS(field##pkts, (val)); \
+        ip4_statistics.field##pkts ++; \
         __IP4_ADD_STATS(field##octets, (val)); \
```

Comment:
What's the problem of the original code? I don't think it changes anything.

Reply:
they are the same and I modified it to make the codes more consistent.